.github/workflows/ci.yml

```diff
@@ -22,20 +22,18 @@ jobs:
           skip: "./docs/Gemfile.lock,./docs/_config.yml,./.github,./.git,./greenboot.spec,./dist"
 
   fmt:
-    name: Rustfmt
+    name: Cargo fmt
     runs-on: ubuntu-latest
+    container: fedora:latest
     steps:
+      - name: Install deps
+        run: |
+          dnf install -y cargo rustfmt
       - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: stable
-          override: true
-      - run: rustup component add rustfmt
       - uses: actions-rs/cargo@v1
         with:
           command: fmt
-          args: --all -- --check
+          args: --check --all
```

Comment:
Looking at the docs for [actions-rs/cargo](https://github.com/actions-rs/cargo), I think this can be simplified to:

```
name: cargo fmt
runs-on: ubuntu-latest
steps:
  - uses: actions/checkout@v3
  - uses: actions-rs/toolchain@v1
    with:
      toolchain: stable
  - uses: actions-rs/cargo@v1
    with:
      command: fmt
      args: --check --all
```